### DISCUSSION TITLE
ValueError: neg_ratio specified, but no bboxes found in scene.

### DISCUSSION THREAD

**Discussion by asoxien** [azavea/raster-vision#1494] [Q&A]
Hi everyone; I already use rasterversion before lastly I'am trying to upgrade to the last version but I get this error and I can't solve it:

  File "/opt/src/rastervision_pytorch_learner/rastervision/pytorch_learner/dataset/object_detection_dataset.py", line 184, in __init__
    'neg_ratio specified, but no bboxes found in scene.')
ValueError: neg_ratio specified, but no bboxes found in scene.

Thank you for your help

**Comment by AdeelH** [comment]
The error means that `neg_ratio` (which controls the ratio at which negative and positive chips are sampled) is set but there are no ground truth bounding boxes in the scene i.e. no possible positive chips. 

As noted in #1430, `ObjectDetectiogeoDataWindowConfig` does not allow `neg_ratio` to be set to `None`. However, I have just fixed this in #1495, so you should be able to set it to `None` if you use the master branch.

If it is possible, I would suggest using to the master branch. If not, you can still work around this by  
- using sliding windows (instead of random windows) 
- excluding this or any other scenes that do not have any ground truth bounding boxes from the training/validation set
- if using docker, manually making the changes in #1495 to the code

**Reply by asoxien** [reply]
great Adeel thank you very much
